introduction.md

```diff
@@ -6,3 +6,13 @@ a codeschool
 Here is the modification for the git course:
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?